sdk/translation/tests.yml

```diff
@@ -23,6 +23,7 @@ stages:
             - PythonVersion=^(?!3.7).*
             - PythonVersion=^(?!3.8).*
             - PythonVersion=^(?!3.9).*
+            - PythonVersion=^(?!3.10).*
```

Comment:
Is this a good opportunity to address the above TODO and switch to specifying the version TO run?

Reply:
Sure! Do you know if there are docs that explain how to do that?

Reply:
The filters are just a regex ([docs](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/job-matrix/README.md#filters)). In this case, I think you could delete all the above negative regex items in favor of only including `3.11`. Either way this filter will need to be updated when the version is bumped.

Reply:
Thanks, those docs are great, bookmarked! I'll update to your suggestion.

Reply:
@benbp is this good to merge?